### PR TITLE
🚧 Profile Backgrounds

### PR DESCRIPTION
**Pull request by pedrofracassi**
- [ ] Make a database schema for backgrounds
- [ ] Edit the user schema and add fields for the current background and purchased ones
- [ ] Fix bottom half cropping (blurred part) - @davipatury 
- [ ] Create API endpoints for...
  - [ ] querying all available backgrounds
  - [ ] querying a user's purchased backgrounds
  - [ ] purchasing a background
  - [ ] equipping a background
  

**Temporary:** Use `s!profile <image name>` to test a background. Backgrounds should be added to `src/assets/png/backgrounds`. When this feature is finished, backgrounds will be pulled from the user's database document, and stored on a separate hosting service (Amazon S3, maybe?)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "booru": "^1.3.4",
     "bufferutil": "^4.0.0",
     "cheerio": "^1.0.0-rc.2",
+    "chroma-js": "^2.0.2",
```
Comment (pedrofracassi) [inline package.json]: I think this shouldn't be here?

**Comment by pedrofracassi** [comment]
What should the database schema look like?

**Comment by metehus** [comment]
can we use #777 for that ?

**Comment by pedrofracassi** [comment]
Closing due to inactivity

**Pull request closed without merging**